boa/cli/mambabuild.py

```diff
@@ -114,7 +124,11 @@ def call_conda_build(action, config, **kwargs):
     """
     recipe = config.recipe[0]
 
-    if action == "test":
+    if action == "output":
+        suppress_stdout()
+        result = api.get_output_file_paths(recipe, config=config, **kwargs)
+        print(result)
```

Comment:
This doesn't print exactly the same way that conda does, does it?

Reply:
The return value is `sorted(list(set(outs)))`